migrations/versions/040ea57b3ea5_initial_migration.py

```diff
@@ -0,0 +1,130 @@
+"""Initial migration
+
+Revision ID: 040ea57b3ea5
+Revises:
+Create Date: 2024-06-19 21:43:15.560972
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+import artemis.db
+
+# revision identifiers, used by Alembic.
+revision = "040ea57b3ea5"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "analysis",
+        sa.Column("id", sa.String(), nullable=False),
+        sa.Column("created_at", sa.DateTime(), server_default=sa.text("NOW()"), nullable=True),
+        sa.Column("target", sa.String(), nullable=True),
+        sa.Column("tag", sa.String(), nullable=True),
+        sa.Column("stopped", sa.Boolean(), nullable=True),
+        sa.Column(
+            "fulltext",
+            artemis.db.TSVector(),
+            sa.Computed("to_tsvector('english', COALESCE(tag, '') || ' ' || COALESCE(target, ''))", persisted=True),
+            nullable=True,
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index("analysis_fulltext", "analysis", ["fulltext"], unique=False, postgresql_using="gin")
+    op.create_index(op.f("ix_analysis_stopped"), "analysis", ["stopped"], unique=False)
+    op.create_index(op.f("ix_analysis_tag"), "analysis", ["tag"], unique=False)
+    op.create_index(op.f("ix_analysis_target"), "analysis", ["target"], unique=False)
+    op.create_table(
+        "report_generation_task",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("created_at", sa.DateTime(), server_default=sa.text("NOW()"), nullable=True),
+        sa.Column("comment", sa.String(), nullable=True),
+        sa.Column("status", sa.String(), nullable=True),
+        sa.Column("tag", sa.String(), nullable=True),
+        sa.Column("language", sa.String(), nullable=True),
+        sa.Column("skip_previously_exported", sa.Boolean(), nullable=True),
+        sa.Column("custom_template_arguments", sa.JSON(), nullable=True),
+        sa.Column("output_location", sa.String(), nullable=True),
+        sa.Column("error", sa.String(), nullable=True),
+        sa.Column("alerts", sa.JSON(), nullable=True),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(op.f("ix_report_generation_task_comment"), "report_generation_task", ["comment"], unique=False)
+    op.create_index(op.f("ix_report_generation_task_status"), "report_generation_task", ["status"], unique=False)
+    op.create_table(
+        "scheduled_task",
+        sa.Column("created_at", sa.DateTime(), server_default=sa.text("NOW()"), nullable=True),
+        sa.Column("analysis_id", sa.String(), nullable=False),
+        sa.Column("deduplication_data", sa.String(), nullable=False),
+        sa.Column("deduplication_data_original", sa.String(), nullable=True),
+        sa.Column("task_id", sa.String(), nullable=True),
+        sa.PrimaryKeyConstraint("analysis_id", "deduplication_data"),
+    )
+    op.create_table(
```

Comment:
tag cannot be in the initial migration. how in that case would we migrate existing DBs? e.g. current Artemis production?

Adding tag should be a separate migration **populating the table from existing task results**